### ISSUE TITLE
Help Text is reading twice using JAWS

### ISSUE THREAD

**Issue by szoomers** [FreedomScientific/standards-support#815]
### Summary

> Brief description of the issue. Please list any specific steps.

When navigating the Details tab of a Record with the JAWS screen reader the Information Icons are announced twice inconsistently. The tooltip of the first Info Icon is not announced twice, but most of the time the second and subsequent third are relayed twice. From this point, it becomes hit or miss whether the tooltip is announced twice. However, when using NVDA none of these issues appear. Each tooltip is read once on focus.

Actual Result:
Information Icons are inconsistently read twice when navigating with JAWS.

Expected Result:
When navigating a Details tab or other page where Information Icons are provided, the screen reader should only relay the Information Icon once on focus regardless of the screen reading technology utilized.

Steps To repro:
Please create a new salesforce org via https://login.salesforce.com/?locale=in
Click on try it for free
Login using the above credentials.
Navigate to the Accounts > Global Mediaaa and select the details tab.
Turn on JAWS
Tab through the details tab and listen to the inconsistency of the Info Icons being announced twice
For further testing review at Cases > Case record and select the Details tab
Now switch to NVDA and repeat steps 5 and 6. None of the Info Icons are repeated twice.

Note:
WCAG Success Criteria: 3.3.2 Labels or Instructions
Helpful W3C recommendations: https://www.w3.org/TR/wai-aria-1.1/#tooltip
Impact: [Limited Impact, Difficult to Finish, or Task Blocking] Difficult to Finish
Code snippet (if applicable): "Marital Status" label Info Icon in Customer Org
﻿<div aria-describedby="tt-for-4874:0" tabindex="0" class="tooltip-trigger uiTooltip" data-aura-rendered-by="4877:0" data-aura-class="uiTooltip" data-proxy-id="aura-pos-lib-4" id="4877:0"><lightning-icon icon-name="utility:info" class="slds-icon-utility-info tooltipIcon slds-button__icon slds-icon_container forceIcon" data-data-rendering-service-uid="1329" data-aura-rendered-by="4873:0" data-aura-class="forceIcon"><span style="--sds-c-icon-color-background: var(--slds-c-icon-color-background, transparent)" part="boundary"><lightning-primitive-icon size="xx-small" variant="inverse"><svg class="slds-icon slds-icon_xx-small" focusable="false" data-key="info" aria-hidden="true" viewBox="0 0 520 520" part="icon"><g><path d="M260 20a240 240 0 100 480 240 240 0 100-480zm0 121c17 0 30 13 30 30s-13 30-30 30-30-13-30-30 13-30 30-30zm50 210c0 5-4 9-10 9h-80c-5 0-10-3-10-9v-20c0-5 4-11 10-11 5 0 10-3 10-9v-40c0-5-4-11-10-11-5 0-10-3-10-9v-20c0-5 4-11 10-11h60c5 0 10 5 10 11v80c0 5 4 9 10 9 5 0 10 5 10 11z"></path></g></svg></lightning-primitive-icon></span></lightning-icon><span role="tooltip" id="tt-for-4874:0" class="tooltip-invisible" data-aura-rendered-by="4879:0">Client's wedding anniversary</span></div>

**Comment by Webhac** [comment]
With JAWS version 2025.2505.43, the redundant announcement of tooltip contents does not seem to occur in Chrome.
